SeaBattle/Sea.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+using System.Security.Cryptography;
+using System.Xml.Linq;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        public List<BaseShip> Ships { get; set; }

```

Comment:
Why this collection is public when you implemented the indexer?

Reply:
Because I am doing checks in BaseShip, and if I do this collection private, it will be unavailable due to its protection level.